core/config/envvar/envvar.go

```diff
@@ -73,18 +73,21 @@ func (e *EnvVar) ParseFrom(get func(string) string) (v interface{}, invalid stri
 	return
 }
 
+// ParseString TODO
```

Comment:
I noticed we're not using the param `v`?

Reply:
It still behaves just like an unnamed return value. In this case they are named just for clarity since `(string, string)` isn't very helpful.